src/components/Project/ProjectNew.js

```diff
@@ -0,0 +1,299 @@
+                  <h4 className='card-title'>New Project</h4>
+                </div>
+                <div className='card-body'>
+                  <br />
+                  <form onSubmit={this.handleSubmit}>
+                    <div className='form-row'>
+                      <div
+                        className='form-group col-sm-12 col-md-6 has-info'
+                        style={{marginTop: '15px'}}>
+                        <label for='inputTitle'>Title:</label>
+                        <input
+                          style={{marginTop: '17px'}}
+                          type='text'
+                          name='title'
+                          className='form-control'
+                          id='inputTitle'
+                          onChange={this.handleChange}
+                          required
+                        />
+                      </div>
+                      <div
+                        className='col-sm-12 col-md-6 has-info'
+                        style={styles.clients}>
+                        <label for='inputClient' className='text-info'>
+                          Client:
+                        </label>
+                        <Select
+                          id='inputClient'
+                          name='clientSelect'
+                          value={this.state.clientSelect}
+                          options={clientOptions}
+                          onChange={this.handleClientChange}
+                          styles={styles.control}
+                          theme={theme => ({
+                            ...theme,
+                            borderRadius: 0,
+                            colors: {
+                              ...theme.colors,
+                              primary25: '#1fbfd7',
+                              primary: '#1fbfd7',
+                            },
+                          })}
+                          required
+                        />
+                      </div>
+                    </div>
+                    <div className='form-row'>
+                      <div
+                        className='form-group col-sm-12 col-md-3 has-info'
+                        style={styles.dates}>
+                        <label for='inputStartDate'>Start Date:</label>
+                        <input
+                          style={{marginTop: '17px'}}
+                          type='date'
+                          name='start_date'
+                          className='form-control'
+                          id='inputStartDate'
+                          onChange={this.handleChange}
+                          required
+                        />
+                      </div>
+                      <div
+                        className='form-group col-sm-12 col-md-3 has-info'
+                        style={styles.dates}>
+                        <label for='inputEndDate'>Start End:</label>
```

Comment:
https://screencast.com/t/vZ9BOr9Tth 
```suggestion
                        <label for='inputEndDate'>End Date:</label>
```